lib/documents.js

```diff
@@ -506,7 +506,7 @@ WithFragments.prototype = {
  * @global
  * @alias Doc
  */
-function Document(id, uid, type, href, tags, slugs, data) {
+function Document(id, uid, type, href, tags, slugs, firstPublicationDate, lastPublicationDate, data) {
```

Comment:
Is camel case really the casing the library is using? I see more train case than camel case see `form_method` in `Form` constructor, the arguments in `Response` constructor etc.